test/basic.js

```diff
@@ -3,22 +3,11 @@
 const toBuffer = require('../')
 const test = require('tape')
 
-const blob = new Blob([new Uint8Array([1, 2, 3])], { type: 'application/octet-binary' })
+const blob = new Blob([new Uint8Array([1, 2, 3])])
 
 test('Basic tests', function (t) {
-  toBuffer(blob, function (err, buffer) {
-    if (err) throw err
+  toBuffer(blob).then(buffer => {
     t.deepEqual(buffer, Buffer.from([1, 2, 3]))
     t.end()
   })
 })
-
-test('Callback error on invalid arguments', function (t) {
-  t.throws(function () {
-    toBuffer({ blah: 1 }, function () {})
-  })
-  t.throws(function () {
-    toBuffer(blob)
-  })
-  t.end()
-})
```

Comment:
Why was this test removed? What happens when an invalid argument like an object is passed in?

Reply:
`Response` is able to accept many things, Typed arrays, ArrayBuffer, strings, Blob/files, FormData, URLSearchParams, ReadableStream, and even DataView and convert it to a ArrayBuffer using `new Response(x).arrayBuffer()`

It would be silly to just restrict it to only Blobs when you can accept many more types of stuff and let it be more loose and be used for more than just Blobs.

anything not understood by the Response constructor will be casted to a string using the `toString` function or secondly `String(x)`

so what happens when you do 
- `await new Response({}).text()` is that you get `"[object Object]"`
- `await new Response(new Date()).text()` yields `"Sat Aug 24 2019 00:33:59 GMT+0200 (centraleuropeisk sommartid)"` (since it has a toSting function)
- `await new Response({toString() { return 'abc'}}).text()` yields `"abc"` even doe it's a plain object

So it's a bit hard to get a error when Response can accept literally anything

So it's no longer just blob-to-buffer, it's now anything to buffer